pyproject.toml

```diff
@@ -75,4 +75,11 @@ xfail_strict = true
 filterwarnings = [
     "ignore:distutils Version classes are deprecated.*:DeprecationWarning:pandas.*",
     "ignore:distutils Version classes are deprecated.*:DeprecationWarning:numpy.*"
+]
+# optimise collection
```

Comment:
Does this make a difference for you locally? I did a quick test, and I don't see a significant difference (was testing with `time pytest --collect-only`)

Reply:
it's pretty marginal - maybe 1 or 2 mins of 30 min runtime runnings tests against all 10 virtual envs